src/propNotEq.js

```diff
@@ -0,0 +1,19 @@
+import { curry, compose, not, propEq } from 'ramda';
+
+/**
+ * Returns true if property of object literal does not equals the given value.
+ *
+ * @func
+ * @category Logic
+ *
+ * @example
+ *
+ * 		R_.propNotEq('a', 1, { a : 1}) // false
+ * 		R_.propNotEq('a', 1, { a : 2}) // true
+ * 		R_.propNotEq('a', 1, {}) // true
+ *
+ * @sig String → a → Object → Boolean
+ */
+const propNotEq = curry(compose(not, propEq));
```

Comment:
ca be simplified to:
const propNotEq = complement(propEq);

Reply:
not really...
this would not work.
```
propNotEq('a')({ a: 1 })
```
Lukas, please introduce pathNotEq as well

Reply:
How it should work? What is problem? This is not right by signature.

I think it works fine :-) 
https://goo.gl/6SjwoS

Reply:
sorry, taking bad